client/v2/internal/offchain/msgSignArbitraryData.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package offchain;
+
+import "cosmos_proto/cosmos.proto";
+import "cosmos/msg/v1/msg.proto";
+import "amino/amino.proto";
+
+// MsgSignArbitraryData defines an arbitrary, general-purpose, off-chain message
+message MsgSignArbitraryData {
+  option (amino.name) = "offchain/MsgSignArbitraryData";
+  option (cosmos.msg.v1.signer) = "signerAddress";
+  // AppDomain is the application requesting off-chain message signing
+  string appDomain = 1;
+  // Signer is the sdk.AccAddress of the message signer
+  string signerAddress = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```

Comment:
```suggestion
  string signer = 2 [(cosmos_proto.scalar) = "cosmos.AddressString"];
```
nit

Reply:
maybe is a good idea to also modify it in the [CIP](https://github.com/cosmos/cips/blob/main/cips/cip-X.md#msgsignarbitrarydata) as well.